specification/pom.xml

```diff
@@ -27,7 +27,7 @@
     <groupId>jakarta.platform</groupId>
     <artifactId>platform-spec</artifactId>
     <packaging>pom</packaging>
-    <version>9-SNAPSHOT</version>
+    <version>9.0-SNAPSHOT</version>
```

Comment:
We decided with Jakarta EE 8 that the platform specification (and web profile) will use a single digit for the release version.  Thus, this should stay at 9-SNAPSHOT.